Dockerfile-default-rocm

```diff
@@ -63,13 +149,71 @@ ARG HOROVOD_WITH_TENSORFLOW=1
 ARG HOROVOD_WITH_PYTORCH=1
 ARG HOROVOD_WITHOUT_MXNET=1
 ARG HOROVOD_GPU_OPERATIONS=NCCL
-ARG HOROVOD_WITHOUT_MPI=1
+ARG HOROVOD_WITHOUT_MPI=0
+ARG HOROVOD_WITH_MPI=1
 ARG HOROVOD_GPU=ROCM
-ARG HOROVOD_WITHOUT_MPI=1
-ENV LD_LIBRARY_PATH=/opt/rocm/lib:/opt/rocm/hip/lib
+ENV LD_LIBRARY_PATH=$LD_LIBRARY_PATH;/opt/rocm/lib:/opt/rocm/hip/lib
+
+#TEST ARGs
+ARG HOROVOD_GPU_ALLREDUCE=NCCL
```

Comment:
I get an error during `build-pytorch13-tf210-rocm56`:

`Cannot specify both HOROVOD_GPU_OPERATIONS and HOROVOD_GPU_ALLREDUCE options. Try unsetting one of these variables and reinstalling`

Horovod docs state that `HOROVOD_GPU_OPERATIONS` already includes ALLREDUCE and BROADCAST, so it's redundant setting these args:  https://horovod.readthedocs.io/en/stable/install_include.html

Reply:
this has been cleaned up so the correct var is used.